### ISSUE TITLE
[FEAT] Add some info regarding "Community" folder path in the installer

### ISSUE THREAD

**Issue by Manochvarma** [MSFS-Mega-Pack/MSFS2020-livery-manager#212]
## Description

Currently when we select some other path/drive for livery installation, it shows error as this:
[https://media.discordapp.net/attachments/748737379212460072/784257251547545620/unknown.png?width=906&height=677](url)
There is no enough info saying which community folder does the installer refer to because when we say _Community_ folder we always refer to the default Microsoft Flight Sim's _Community_ folder. 
## Solution

When we create a folder name _Community_ in the folder we wanted to install the livery (other than the _default MSFS drive_), the installer identify it without giving error. So, there should be a sample to show how the community folder should be created.

Sample of mine after adding the community folder in the drive I wish to install the liveries
[https://media.discordapp.net/attachments/748737379212460072/784258153314910228/unknown.png](url)
Example, something like this:

**Default MSFS path**
[Default MSFS path]\Community
or
**Other drive folder paths:**
[Your Drive Path (D, E, F or else) Drive]\Community

### Why is this the best solution?

By having this simple sample, users will not be confused why the installer not allowing us to change the path.

**Comment by davwheat** [comment]
Hmmm you have a good point here.

What do you think about a simple documentation website on GitHub Pages for things like this.

We can then link to the site from the manager, adding a "More info" link to the end of that error text.

Lemme know your opinion and any other ideas you have.

**Comment by Manochvarma** [comment]
I don't think that will work as expected because people wanted a prompt solution rather than searching for it in another link or note files. If you check the 1st image above, there is an error showing in red _Please make sure you have selected your community folder_. I think below that line, we can add sample of the community path as shown above in my explanation above.

_Other drive folder paths:_
[Your Drive Path (D, E, F or else) Drive]\ **Community** 
*Make sure you have created a community folder in the folder you wish to save the liveries.

Bolding/Highlighting the community word in other colours will make them easier to understand that the community folder must be there.

Since the default _MSFS community folder_ can be found by the installer itself with the registry entry, I think better if we can put a simple sentence asking the users to make sure to create their own community folder at the folder they wish to install the liveries. 

Let me know what your point of view.

**Comment by davwheat** [comment]
What situation would people not install to the normal Community Packages folder?

PS: There's no registry key for MSFS (grr). We have to manually snoop into locations where the game is normally installed, then find a text file with a line pointing to the packages folder.

**Comment by Manochvarma** [comment]
> What situation would people not install to the normal Community Packages folder?
> 
> PS: There's no registry key for MSFS (grr). We have to manually snoop into locations where the game is normally installed, then find a text file with a line pointing to the packages folder.

I never know there is no registry entry for this sim.

For me, the only situation is when we don't have enough space on the drive we have installed MSFS.

**Comment by Khitiara** [comment]
I have the liveries installed elsewhere for organizational and disk space reasons, have the liveries in a subfolder of my addon storage and linked into the actual community folder - also allows disabling or re-enabling liveries without having to redownload
